server.js

```diff
@@ -44,6 +44,8 @@ app.use('/public', express.static(__dirname + '/public'));
 app.use('/public', express.static(__dirname + '/govuk_modules/govuk_template/assets'));
 app.use('/public', express.static(__dirname + '/govuk_modules/govuk_frontend_toolkit'));
 app.use('/public/images/icons', express.static(__dirname + '/govuk_modules/govuk_frontend_toolkit/images'));
+app.use('/public/images', express.static(__dirname + '/hmrc_modules/images'));
```

Comment:
We only need the one route don't we? Similar to above...

`app.use('/public', express.static(__dirname + '/hmrc_modules'));`